vivo-project/VIVO
# Removed files moved to Vitro repository

# What does this pull request do?
Removed files that were moved to Vitro repo

# How to test
* Build and install VIVO
* Set languages.selectableLocales = en_US, de_DE and RDFService.languageFilter = true in runtime.properties
* Verify that locale selector is shown
* Verify that /templates/freemarker/edit/forms/css/autocomplete.css file is accessible ([example](https://vivo.tib.eu/vivorc/templates/freemarker/edit/forms/css/autocomplete.css))


# Interested parties
@chenejac
